microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -1,150 +1,280 @@
+
+Cons: 
+*  You need to manage the WebView2 Runtime yourself.  The WebView2 Runtime isn't automatically updated on clients, so to use the latest WebView2 APIs, you must periodically update your app together with the updated WebView2 Runtime.
+*  More disk space is required on the client, if there are multiple WebView2 apps installed.
+*  The Fixed Version WebView2 Runtime can't be installed by using an installer.
+
+
+<!-- ====================================================================== -->
+## Understanding the WebView2 Runtime
+
+The WebView2 Runtime is the backing web platform for WebView2 apps.  When distributing your WebView2 app, make sure that the WebView2 Runtime is present on the client machine.  This requirement applies to both the Evergreen and Fixed Version distribution modes.
+
+If you use the Fixed Version distribution mode, you can skip the remainder of this section, which applies to the Evergreen distribution mode.
 
-Do not take a dependency on the browser because:  
+<!-- orig start of this section: -->
+The WebView2 Runtime is a redistributable runtime and serves as the underlying (or _backing_) web platform for WebView2 apps.  The concept is similar to Visual C++ or the .NET Runtime for C++/.NET apps.  The WebView2 Runtime contains modified Microsoft Edge \(Chromium\) binaries that are fine-tuned and tested for apps.  After the WebView2 Runtime is installed, it doesn't appear as a user-visible browser app.  For example, a user does not have a browser desktop shortcut or an entry in the **Start** menu.
 
-*   Microsoft Edge \(Chromium\) is not guaranteed to be present on all user devices.  For example, devices disconnected from Windows Update or not managed by Microsoft directly \(a large portion of the Enterprise and EDU market\) may not have the browser.  Allowing you to distribute the WebView2 Runtime avoids taking a dependency on the browser as a pre-requisite for apps.  
-*   Browsers and apps have different use cases, and so taking a dependency on a browser may have unintended side-effects on your apps.  For example, IT admins may version-control the browser for internal website compatibility.  The WebView2 Runtime allows apps to stay evergreen while browser updates are being actively managed.  
-*   As opposed to the browser, the Runtime is developed and tested for app scenarios and in some cases may include bug fixes not yet available in the browser.  
+### Runtime or browser support during development or production
+
+During development and testing, a WebView2 app can use either option as the backing web platform:
+*   The WebView2 Runtime.
+*   A non-stable (Insider) Microsoft Edge \(Chromium\) browser channel.  Use this approach to test your app for forward-compatibility, so that you know if a breaking change is coming that will require updating your app.
```

Comment:
Global in distribution.md & versioning.md: change "non-stable" or "Insider" to variants of "Microsoft Edge preview channels", per standard usage in this repo.